src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -8118,29 +8133,63 @@ void checkWellKnown(WellKnownMember member)
 
             var lookupResult = LookupResult.GetInstance();
```

Comment:
>lookupResult [](http://example.com/codeflow?start=16&length=12)

It doesn't look like ```lookupResult``` is used in this function in any meaningful way. If ```TryFindIndexOrRangeImplicitIndexer``` needs it, it should be able to allocate ```LookupResult``` itself and manage its lifetime.